source/administrator-guide/configuring-the-catalog/css-configuration.rst

```diff
@@ -0,0 +1,23 @@
+.. _csw-configuration:
+
+Configuring CSS & Style
+#######################
+
+To get to the CSS & Style configuration, you must be logged on as administrator. 
+Open 'Settings' from the Administration page and select `CCS & Style` configuration.
+
+.. figure:: img/css-screen.png
+
+This page facilitates to change various colors of screen elements. Mind that 'background' refers to the background color of 
+elements and 'color' to the font color used in the element. Above configuration will result in below vizualisation:
```

Comment:
This page allows you to change various colors of page elements. Note that 'background' refers to the background color of elements, and 'color' to the font color used in the element.  The  above configuration will result in the vizualisation below: